scripts/dataController.js

```diff
@@ -0,0 +1,10 @@
+(function(module) {
+  var dataController = {};
+  dataController.index = function() {
+    $('.tab-content').hide();
```

Comment:
I would encourage this type of functionality be moved to something like an `init` method in the view, called here inside the index, and then issuing a callback/next as well.

This is a consistent comment for your controller files as a whole.